common/app/views/support/AmpAd.scala

```diff
@@ -0,0 +1,43 @@
+package views.support
+
+import model.Tag
+import model.Article
+import play.api.libs.json.Json
+
+object AmpAd {
+  private def grabLastFragmentOfId(items: Seq[Tag]) = {
```

Comment:
would be `grabLastFragmentsOfIds` but as below I'm guessing it is more like `turnTagsIntoAdFlags` or something?